code/src/components/AlbumList/AlbumList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Album } from '../Album/Album';
+
+export const AlbumList = (props) => {
+  console.log(props)
+  return (
+    <section className="AlbumList">
+      {props.data.albums.items.map((singleAlbum) => (
```

Comment:
Have I understood it correctly that AlbumList.js is only a component for filtering the data from data.json?